code/unclassified/biggest_of_n_numbers/biggest_of_n_numbers.py

```diff
@@ -0,0 +1,6 @@
+elements = []
+n = int(input("Enter number of elements:"))
+for i in range(1, n + 1):
+  elements.append(int(input("Enter element:")))
+elements.sort()
+print("Largest element is : ", elements[n - 1])
```

Comment:
You don’t even need an array, you could just keep track of the largest number constantly.

Reply:
@arnavb That can be one of the logic but i think this is much simpler one .